### DISCUSSION TITLE
API Error

### DISCUSSION THREAD

**Discussion by muttmutt** [TheAxeDude/ZiniGo#45] [Q&A]
I'm getting the following error downloading my Zinio inventory on MacOS:

`{"status":false,"error":{"message":"Route not found in ApiRouter","internal_code":"APPERR13131313"}}`

Any thoughts?

**Comment by onigame** [comment]
I'm getting the exact same error, on a Windows machine.

**Comment by onigame** [comment]
It looks like what happened is that Zinio changed their URL scheme.  According to the source code here, it is expecting

`https://www.zinio.com/api/reader/content?issue_id=[NUMBER]`

but currently zinio uses

`https://www.zinio.com/reader/readsvg/[NUMBER]`

I note that when I navigate to https://www.zinio.com/api/reader I get the same response:

`{"status":false,"error":{"message":"Route not found in ApiRouter","internal_code":"APPERR13131313"}}`

Unfortunately I can't figure out how to build go files, otherwise I'd try to fix this.

**Reply by GenesisDCarmen** [reply]
Thank you! Maybe with this information @TheAxeDude can generate a new .exe file

**Comment by GenesisDCarmen** [comment]
Same error :(

Fetching page:1
{"status":false,"error":{"message":"Route not found in ApiRouter","internal_code":"APPERR13131313"}}
Terminating the application...
